assets/css/app.scss

```diff
@@ -0,0 +1,24 @@
+/* This file is for your main application CSS */
+@import "./phoenix.scss";
+@import "bulma";
+
+.hidden{
+  visibility: hidden;
+}
```

Comment:
Bulma has "is-invisible" helper, so there should be no need of this
https://bulma.io/documentation/helpers/visibility-helpers/